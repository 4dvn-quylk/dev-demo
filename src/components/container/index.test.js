/* eslint-disable testing-library/prefer-screen-queries */
import '@testing-library/jest-dom';
import { render, screen } from '@testing-library/react'; // (or /dom, /vue, ...)
import { Container } from './index';

describe('Test Container component', () => {
  test('should render', () => {
    render(<Container />);
    const container = screen.getByTestId('c-container');
    expect(container).toBeInTheDocument();
  });
  test('should have children', () => {
    render(<Container>hello</Container>);
    expect(screen.getByText('hello')).toBeTruthy();
  });
  test('should have no children', () => {
    render(<Container />);
    expect(screen.queryByText('hello')).toBeFalsy();
  });
});
