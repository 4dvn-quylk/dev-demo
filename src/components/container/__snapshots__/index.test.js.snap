// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Container component should render 1`] = `
<div>
  <div
    class="c-container"
    data-testid="c-container"
  />
</div>
`;
